tests/lca/imagebasedupgrade/cnf/upgrade-talm/tests/rollback-after-failed-upgrade-test.go

```diff
@@ -111,52 +83,19 @@ var _ = Describe(
 
 		It("Rollback after a failed upgrade", reportxml.ID("69054"), func() {
 
-			By("Creating, enabling ibu pre-prep CGU and waiting for CGU status to report completed", func() {
-				prePrepCguBuilder := cgu.NewCguBuilder(TargetHubAPIClient,
-					tsparams.PrePrepCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.PrePrepPolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				prePrepCguBuilder.Definition.Spec.Enable = ptr.To(true)
-
-				prePrepCguBuilder, err = prePrepCguBuilder.Create()
-				Expect(err).NotTo(HaveOccurred(), "Failed to create pre-prep CGU.")
-
-				_, err = prePrepCguBuilder.WaitUntilComplete(10 * time.Minute)
-				Expect(err).NotTo(HaveOccurred(), "Pre-prep CGU did not complete in time.")
-			})
-
-			By("Creating, enabling ibu prep CGU and waiting for CGU status to report completed", func() {
-				prepCguBuilder := cgu.NewCguBuilder(TargetHubAPIClient,
-					tsparams.PrepCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.PrepPolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				prepCguBuilder.Definition.Spec.Enable = ptr.To(true)
-
-				prepCguBuilder, err = prepCguBuilder.Create()
-				Expect(err).NotTo(HaveOccurred(), "Failed to create prep CGU.")
-
-				_, err = ibu.WaitUntilStageComplete("Prep")
-				Expect(err).NotTo(HaveOccurred(), "error waiting for prep stage to complete")
-
-				_, err = prepCguBuilder.WaitUntilComplete(25 * time.Minute)
-				Expect(err).NotTo(HaveOccurred(), "Prep CGU did not complete in time.")
-			})
-
-			By("Creating, enabling ibu upgrade CGU, and waiting for node rebooted into stateroot B", func() {
-
-				By("Creating and enabling ibu upgrade CGU")
+			By("Creating Prep->Upgrade->FinalizeUpgrae IBGU and waiting for node rebooted into stateroot B", func() {
 
-				upgradeCguBuilder := cgu.NewCguBuilder(TargetHubAPIClient,
-					tsparams.UpgradeCguName, tsparams.IbuCguNamespace, 1).
-					WithCluster(tsparams.TargetSnoClusterName).
-					WithManagedPolicy(tsparams.UpgradePolicyName).
-					WithCanary(tsparams.TargetSnoClusterName)
-				upgradeCguBuilder.Definition.Spec.Enable = ptr.To(true)
+				newIbguBuilder = ibgu.NewIbguBuilder(TargetHubAPIClient,
+					tsparams.IbguName, tsparams.IbguNamespace).
+					WithClusterLabelSelectors(tsparams.ClusterLabelSelector).
+					WithOadpContent("oadp-cm", "ztp-group").
```

Comment:
Ditto, we need these loaded as config parameters.

Reply:
Updated- thank you.

Reply:
The NS probably shouldn't be hardcoded to ztp-group.
Look at this thread: https://redhat-internal.slack.com/archives/C05JHD9QYTC/p1726797433687639

Reply:
@achuzhoy Thanks- this was changed to use the oadp cm name/namespace variable passed from the CI profile